src/sql/workbench/browser/modelComponents/toolbarContainer.component.ts

```diff
@@ -47,14 +53,22 @@ export class ToolbarItem {
 				</div>
 			</div>
 			</ng-container>
+			<a #moreActionsButton class="moreActions action-label codicon toggle-more" role="button" tabindex="0" aria-haspopup="true"> </a>
```

Comment:
So what we should really be doing here is re-using the actual toolbar component we already define in sql/base/ui. I know that's a good chunk more work but duplicating the code like this just makes it more likely we're going to have issues down the line with fixes/improvements being made in one but the other isn't also fixed/updated.

Reply:
Especially when you see stuff like the duplicated resizing code. That's definitely a huge red flag and should be avoided at all costs.